### PR TITLE
Add JetBrains Space.app 2020.1.8

### PR DESCRIPTION
**Pull request by tiulpin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

The app is in Beta phase now. I will add [appcast](https://data.services.jetbrains.com/products/releases?code=SP) to check versions (and version + checksum), when it's available.
FYI: I'm not from JetBrains Space development team, just a user, who wants to install the app via Homebrew Cask.

### CODE REVIEW

File: Casks/jetbrains-space.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "jetbrains-space" do
+  version :latest
+  sha256 :no_check
+
+  url "https://download.jetbrains.com/space/jetbrains-space.dmg"
+  name "JetBrains Space"
+  desc "The integrated team environment"
```
Comment (reitermarkus) [inline Casks/jetbrains-space.rb:7]: This sounds more like a slogan than a description. I don't know what this is just from reading this.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/jetbrains-space.rb (hunk 1 above)
Comment (tiulpin) [reply Casks/jetbrains-space.rb:7]: OK, I'll remove the description then. Will it be OK?

Comment (tiulpin) [reply Casks/jetbrains-space.rb]: Sorry, the description is needed, will try to add it ASAP

**Comment by tiulpin** [comment]
Some casks (e.g. https://github.com/Homebrew/homebrew-cask/blob/master/Casks/jira-client.rb) don't have descriptions. It's difficult for me to write the good description in one line for that cask. Is it OK to add package without a description?

**Comment by reitermarkus** [comment]
It's required for new casks and encouraged for existing ones so we eventually have descriptions for all casks.

**Comment by tiulpin** [comment]
OK, added the most correct description I could think of.

**Comment by tiulpin** [comment]
@reitermarkus thank you for the edit! Can this PR be accepted now?

**Review by reitermarkus — APPROVED** [review]

**Pull request merged**